src/dictionary.c

```diff
@@ -82,7 +83,10 @@ static void dictionary_name_value_destroy(DICTIONARY *dict, NAME_VALUE *nv) {
 	}
 	pthread_rwlock_unlock(&dict->rwlock);
 
-	free(nv->value);
+  // FIX: strdup() uses malloc to allocate new string. Must be freed.
+  free(nv->name);
```

Comment:
thanks!
We are lucky... dictionary.c is not used yet. I wrote it to replace all the indexes in rrd.c appconfig.c, etc but never completed the task.